libcontainer/configs/validate/rootless_test.go

```diff
@@ -93,13 +93,13 @@ func TestValidateRootlessEUIDMountUid(t *testing.T) {
 	}
 
 	config.Mounts[0].Data = "uid=2"
-	config.UidMappings[0].Size = 10
+	config.UIDMappings[0].Size = 10
 	if err := Validate(config); err != nil {
 		t.Errorf("Expected error to not occur when setting uid=2 in mount options and UidMapping[0].size is 10")
 	}
 
 	config.Mounts[0].Data = "uid=20"
-	config.UidMappings[0].Size = 10
+	config.UIDMappings[0].Size = 10
 	if err := Validate(config); err == nil {
 		t.Errorf("Expected error to occur when setting uid=20 in mount options and UidMapping[0].size is 10")
```

Comment:
I think you misunderstand my thought. Next time I will write it completely.
s/UidMapping/UIDMapping/g
s/GidMapping/GIDMapping/g
I search them in the whole file, find 4 locations needs to be fixed.
I think these are related to your PR.

Reply:
and UidMapping[0].size is 10

Reply:
> s/Uid/UID/ You can search them globally. There are many Uid and Gid.

Sorry, It's my fault, I should say it clearly.
I thought you know `Uid` means `UidMapping`, I didn't want to type so many characters, and I would not let you corrent any `Uid` in this PR. I should not be lazy to type `Mapping` after `Uid`.

I should write like this:
```
s/UidMapping/UIDMapping/
You can search them globally.
There are many UidMapping and GidMapping.
```

Reply:
I search them globally, only find 4 locations remained needs to be fixed in`libcontainer/configs/validate/rootless_test.go`.
You don't need to update other files.

Reply:
> 🤯 I think there is a _quid pro quo_ here. So, you are asking me to use `UID` rather than `UIDMappings` (and the same for `GID`), right?

I only wanted to ask you change `UidMapping` -> `UIDMapping`, `GidMapping` -> `GIDMapping`. I didn't want to search by myself at that time, so I asked you search by yourslef. I had no othere meanings.

Reply:
> I do not understand your comment but I normally cleaned everything with:

You don't need to do this.

Reply:
I think you still misunderstand my meaning. Let's close this conversation, open a new conversation.